worldedit-core/src/main/java/com/sk89q/worldedit/command/ClipboardCommands.java

```diff
@@ -265,4 +266,58 @@ public void clearClipboard(Actor actor, LocalSession session) {
         session.setClipboard(null);
         actor.printInfo(TranslatableComponent.of("worldedit.clearclipboard.cleared"));
     }
+
+
+    @Command(
+        name = "/revolve",
+        desc = "Revolve the selection around a vertical axis"
+    )
+    @CommandPermissions("worldedit.revolve")
+    void revolve(Actor actor, LocalSession session, EditSession editSession, @Selection Region region,
+                 @Arg(desc = "The number of pastes")
+                    int pasteCount,
+                 @ArgFlag(name = 'm', desc = "Set the source mask, non-matching blocks are not revolved")
+                    Mask mask,
+                 @Switch(name = 'r', desc = "Perform revolutions in reverse (counter-clockwise)")
+                    boolean reverse,
+                 @Switch(name = 'e', desc = "Copy entities")
+                    boolean copyEntities,
+                 @Switch(name = 'b', desc = "Copy biomes")
+                    boolean copyBiomes) throws WorldEditException {
+        checkRegionBounds(region, session);
+        checkCommandArgument(pasteCount >= 2, TranslatableComponent.of("worldedit.revolve.too-few-pastes"));
+
+        // Copy the selection into a clipboard
+        BlockArrayClipboard clipboard = new BlockArrayClipboard(region);
+        clipboard.setOrigin(session.getPlacementPosition(actor));
+        ForwardExtentCopy copy = new ForwardExtentCopy(editSession, region, clipboard, region.getMinimumPoint());
+        copy.setCopyingEntities(copyEntities);
+        copy.setCopyingBiomes(copyBiomes);
+
+        if (mask != null) {
+            copy.setSourceMask(mask);
+        }
+        Operations.complete(copy);
+
+        double perPasteDegrees = 360.0 / pasteCount;
+        ClipboardHolder holder = new ClipboardHolder(clipboard);
+
+        // Now paste it multiple times, rotating each time
+        for (int i = 1; i < pasteCount; i++) {
+            AffineTransform transform = new AffineTransform();
+            transform = transform.rotateY((reverse ? 1 : -1) * perPasteDegrees * i);
```

Comment:
Can we do this as:
```suggestion
            AffineTransform transform = new AffineTransform().rotateY((reverse ? 1 : -1) * (360 * i) / (double) pasteCount);
```
I think we're much less likely to lose precision as much scaling up first, then scaling down; versus the other way around which at large numbers might start to get very small `perPasteDegrees`. This way, even at hundreds of thousands of pastes, it will still be quite accurate, as we keep integers up to the division.

Also, I don't think reassigning to this variable is very useful, so just in-lining the initialization.